ants_test.go

```diff
@@ -973,3 +973,14 @@ func TestReleaseTimeout(t *testing.T) {
 	err = pf.ReleaseTimeout(2 * time.Second)
 	assert.NoError(t, err)
 }
+
+func TestDefaultPoolReleaseTimeout(t *testing.T) {
+	for i := 0; i < 5; i++ {
```

Comment:
Since the default pool has been released by the previous tests, you should call `Reboot()` before starting your new tests.